From f85a94e84f01a4762a7ebaa3a0058f29c93c5cc2 Mon Sep 17 00:00:00 2001
From: Rogerio Guerra Borin <rogerio.borin@toradex.com>
Date: Thu, 10 Oct 2024 00:39:52 -0300
Subject: [PATCH 10/10] toradex: show message if CLI access allowed
 (downstream)

Before, a message was shown only when the device was closed and the CLI
access was disabled by the CLI protection feature. Now we show a message
also when the device is open which is helpful when someone wants to
ensure the protection is compiled in without having to close the device.

Upstream-Status: Inappropriate [TorizonCore specific]

Signed-off-by: Rogerio Guerra Borin <rogerio.borin@toradex.com>
---
 common/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/main.c b/common/main.c
index 3662c0361e7..28eb98d3bda 100644
--- a/common/main.c
+++ b/common/main.c
@@ -60,7 +60,7 @@ void main_loop(void)
 
 	s = bootdelay_process();
 #if CONFIG_IS_ENABLED(TDX_CLI_PROTECTION)
-	if (!tdx_cli_access_enabled())
+	if (!tdx_cli_access_enabled(1))
 		tdx_secure_boot_cmd(s); 	/* no return */
 #endif
 	if (cli_process_fdt(&s))
-- 
2.25.1

